transit_odp/publish/templates/publish/revision_review/review_line_metadata.html

```diff
@@ -118,6 +118,39 @@ <h1 class="govuk-heading-xl app-!-mb-4 dont-break-out">
         <div class="govuk-grid-column-one-third">
             {% include 'publish/dataset_detail/related_links.html' %}
         </div>
+
+        <div class="govuk-grid-column-full">
+            <h2 class="govuk-heading-m">Timetable</h2>
+            <h2 class="govuk-heading-m">Filter by</h2>
+            <div class="govuk-form-group  govuk-!-width-one-quarter govuk-!-padding-0  ">
+            <label class="govuk-label" for="timetable_date">
+                Date
+            </label>
+            <span id="timetable_date-hint" class="govuk-hint ">
+                For example: 21/11/2014
+            </span>            
+            <input type="text"            
+            id="timetable_date" name="start" style="width: 90%;" class="govuk-input" aria-describedby="timetable_date-hint"
+            value={{curr_date}} data-gtm-form-interact-field-id="0" />
+            <img id="timetable_date_calendar"
```

Comment:
On click event is not displaying the calendar

Reply:
as the image needs to be rebuilt or need to install the new package of air-date picker